storage_controller/migrations/2024-08-23-102952_safekeepers/up.sql

```diff
@@ -0,0 +1,17 @@
+-- very much a copy of cplane version, except that the projects_count is not included
+CREATE TABLE safekeepers (
+	-- the surrogate identifier defined by control plane database sequence
+	id BIGINT PRIMARY KEY,
+	created_at TIMESTAMPTZ NOT NULL DEFAULT now(),
+	updated_at TIMESTAMPTZ NOT NULL DEFAULT now(),
+	region_id TEXT NOT NULL,
+	version BIGINT NOT NULL,
+	-- the natural id on whatever cloud platform
+	instance_id TEXT UNIQUE NOT NULL,
```

Comment:
Discussed a little bit out of band. I don't think we need this.

Reply:
It has been removed in [2c142f1](https://github.com/neondatabase/neon/pull/8879/commits/2c142f14f7ba9bd9a178ae96723d196330eb34ba)